src/components/DatePicker/RangePicker.jsx

```diff
@@ -21,12 +35,23 @@ export const RangePicker = ({
   onChange,
   label,
   placeholder,
+  labelCompareWith,
+  labelCompareDivider,
+  labelLastPeriod,
+  compareLabels,
   ...props
 }) => {
   const wrapRef = useRef()
   const [isExpanded, setExpanded] = useState(false)
   const [isSettingEnd, setSettingEnd] = useState(false)
   const [currentHover, setCurrentHover] = useState(null)
+  const [isComparing, setComparing] = useState(false)
+  const [compareMethod, setCompareMethod] = useState(
+    compareLabels?.[compareMethods.lastPeriod]
+  )
+
+  const value = multiple ? rawValue?.[0] : rawValue
+  const compareValue = multiple ? rawValue?.[1] : null
```

Comment:
```suggestion
  const [value, compareValue] = ... (multiple ? rawValue : [rawValue,null])

```

Reply:
It's not a guaranteed that when multiple = true, rawValue will be an array .. so adding that extra check makes the line more unreadable 

I prefer it this way